src/parser.ts

```diff
@@ -2,7 +2,7 @@ import * as ts from 'typescript';
 import { Dependency, ParsedOutput } from './model.js';
 import * as path from 'path';
 
-const decorators = ['Component', 'Directive', 'Pipe', 'Service'];
+const decorators = ['Component', 'Directive', 'Pipe', 'Injectable'];
```

Comment:
NIce catch. I think that this is exactly the reason why I should add tests ASAP :)